custom_components/tuya_local/devices/bauhn_5way_powerstrip_usb_AP5W-0624.yaml

```diff
@@ -0,0 +1,214 @@
+    category: config
+    translation_key: timer
+    dps:
+      - id: 9
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    name: Timer 2
+    category: config
+    translation_key: timer
+    dps:
+      - id: 10
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    name: Timer 3
+    category: config
+    translation_key: timer
+    dps:
+      - id: 11
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    name: Timer 4
+    category: config
+    translation_key: timer
+    dps:
+      - id: 12
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    name: Timer 4
+    category: config
+    translation_key: timer
+    dps:
+      - id: 13
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: sensor
+    name: Current
+    category: diagnostic
+    class: current
+    dps:
+      - id: 18
+        name: sensor
+        type: integer
+        class: measurement
+        force: true
+        unit: mA
+      - id: 23
+        type: integer
+        name: calibration
+        optional: true
+  - entity: sensor
+    name: Power
+    category: diagnostic
+    class: power
+    dps:
+      - id: 19
+        name: sensor
+        type: integer
+        class: measurement
+        force: true
+        unit: W
+        mapping:
+          - scale: 10
+      - id: 24
+        type: integer
+        name: calibration
+        optional: true
+  - entity: sensor
+    name: Voltage
+    category: diagnostic
+    class: voltage
+    dps:
+      - id: 20
+        name: sensor
+        type: integer
+        class: measurement
+        force: true
+        unit: V
+        mapping:
+          - scale: 10
+      - id: 22
+        type: integer
+        name: calibration
+        optional: true
+  - entity: select
+    name: Initial state
+    icon: "mdi:toggle-switch"
+    category: config
+    dps:
+      - id: 38
+        type: string
+        name: option
+        mapping:
+          - dps_val: "on"
+            value: "on"
+          - dps_val: "off"
+            value: "off"
+          - dps_val: "memory"
+            value: Last state
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 41
+        type: boolean
+        name: lock
+        optional: true
+  - entity: sensor
```

Comment:
I don't think these last three should be sensors. Attributes on another entity (Perhaps Timer 1, since they are timer related) is enough to make them available without having them cluttering the default UI and entity list.

Reply:
@make-all Thanks very much for the feedback, I've updated the other changes but wasn't sure what the correct syntax would be to make these ones as attributes of timer 1, is there any guidance you can provide on this one?

Reply:
Something like the way these three are handled in [aubess_2gang_switch](https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/devices/aubess_2gang_switch.yaml#L12) for example (there they are on switch 1, mainly because the primary entity usually ends up with left overs, but since these are all timing related, putting them on a timer also makes sense).  As long as you avoid names with special meaning for the entity, extra attributes just end up as read-only values in the state attributes for the entity.

Reply:
@make-all Thanks very much, I have updated the YAML to have those three sensors as attributes of timer 1 and all seems to be working 😊